modules/store/spec/store.spec.ts

```diff
@@ -187,7 +187,6 @@ describe('ngRx Store', () => {
 
       dispatcher.ngOnDestroy();
 
-      expect(storeSubscription.closed).toBe(true);
```

Comment:
@MikeRyanDev this test was failing as the `storeSubscription` is no longer closed. It didn't show up until I upgraded RxJS. Is this a change in behavior?